publisher/product/ontology/build.sh

```diff
@@ -495,6 +496,51 @@ function ontologyBuildIndex () {
 	return $?
 }
 
+function ontologyCreateMergedFiles() {
+
+  require source_family_root || return $?
+  require tag_root || return $?
+
+  logStep "ontologyCreateMergedFiles"
+
+  pushd "${tag_root:?}" >/dev/null || return $?
+
+  local -r maxParallelJobs=1
+  local numberOfParallelJobs=0
+
+  log "Running ${maxParallelJobs} 'merge-imports' jobs in parallel:"
+
+  # include only the RDF files with the ontologies listed in the source "catalog-v001.xml"
+  for rdfFile in $(cat "${source_family_root}"/catalog-v001.xml 2>/dev/null | xml c14n 2>/dev/null | xml sel -T -t -v '//@uri' -n 2>/dev/null | grep -i '\.rdf') ; do
+    ontologyIsInTestDomain "${rdfFile}" || continue
+    #isProductOntology "${rdfFile}" || continue
```

Comment:
@mereolog whether only ontologies with the correct "base URL" should be taken into account when generating _Merged_ files?

The question is, does it make sense to generate _Merge_ files for ontologies that will **not** be _dereferenceable_?

For example, the ontology in [idmp/CMNS/Documents.rdf](https://github.com/edmcouncil/idmp/blob/master/CMNS/Documents.rdf#L28) has:
`	<owl:Ontology rdf:about="https://www.omg.org/spec/Commons/Documents/">`

So for this ontology to be "dereferenceable", it should be published under the URL:

`https://www.omg.org/spec/Commons/Documents/`

Instead, this ontology will be published under the URL:

`https://spec.pistoiaalliance.org/idmp/ontology/CMNS/Documents/`

i.e. the "base URL" for this ontology is `https://spec.pistoiaalliance.org/idmp/ontology/` as indicated by _owl: Ontology_ in the [DEV_SPEC](https://github.com/edmcouncil/idmp/blob/master/AboutIDMPDev.rdf#L22) file for this repository: `<owl:Ontology rdf:about="https://spec.pistoiaalliance.org/idmp/ontology/AboutIDMPDev/">`

Reply:
1. Use "base URL" to filter out ontologies out of scope.
2. Rename function "isProductOntology" to "isOntologyInScope"

Reply:
See [commit 6210cc0: createMergedFile: only use ontologies that satisfy "isIRIInScope" && …](https://github.com/edmcouncil/ontology-publisher/pull/104/commits/6210cc02fa2a7d74647ca390a2fdfaec36e27101)